Content.Server/VendingMachines/VendingMachineSystem.cs

```diff
@@ -435,6 +438,12 @@ public override void Update(float frameTime)
                     }
                 }
             }
+            var disabled = EntityQueryEnumerator<EmpDisabledComponent, VendingMachineComponent>();
+            while (disabled.MoveNext(out var uid, out _, out var comp))
+            {
+                if (_random.Prob(0.15f * frameTime)) // more lag friendly
+                    EjectRandom(uid, true, false, comp);
+            }
```

Comment:
This should just tick every second as this will be tickrate dependent between 30tps / 60tps for example.

Reply:
it shouldn't be tick rate dependent because i multiply constant by frameTime
and i don't know how to make it tick every second without adding new field to vending machine component

Reply:
Probabilities can't be summed that way and get similar outcomes.

e.g. if you have a probability of 1 and multiply it by a frametime of 0.016 suddenly it's not a probability of 1 anymore. Same with 0.15 ticking once per second doesn't equal 0.0024 ticking every 0.016 seconds

Reply:
made it eject items every 5x the eject delay seconds